README.md

```diff
@@ -94,6 +94,30 @@ Raise an exception if not successful with `run!`:
 Scmd.new("cd /path/that/does/not/exist").run! #=> Scmd::Command::Failure
 ```
 
+### Environment variables
+
+Pass environment variables:
+
+```ruby
+cmd = Scmd.new("echo $TEST_VAR", {
+  :env => 'TEST_VAR' => 'hi'
+})
+```
+
+### Process spawn options
+
+Pass options:
+
+```ruby
+reader, writer = IO.pipe
+cmd = Scmd.new("echo test 1>&#{writer.fileno}", {
+  :options => { writer => writer }
+})
+reader.gets # => "test\n"
+```
+
+For all the possible options see [posix-spawn](https://github.com/rtomayko/posix-spawn#status) and [Process#spawn](http://ruby-doc.org/core/Process.html#method-c-spawn).
```

Comment:
@jcredding we discussed *not* linking to the ruby doc docs since posix spawn doesn't allow everything those docs say.  Since we are using posix spawn, we should just use their docs to avoid confusion.